Wrappers.h

```diff
@@ -0,0 +1,50 @@
+struct CompresssorWrapper
```

Comment:
```
Don't forget the '#pragma once'
```
```
 4) Put all of your Wrapper classes in a single Wrappers.h file
         if you implemented your wrapper class functions in-class, you'll need to move them to Wrappers.cpp
```
You implemented all of your wrapper class functions in-class. 
you need to move the implementations to `Wrappers.cpp`
- [ ] revise